src/main/resources/application.properties

```diff
@@ -1,4 +1,4 @@
 PSQL_URL=jdbc:postgresql://localhost:5432/quizmates
 PSQL_USERNAME=postgres
-PSQL_PASSWORD=root
+PSQL_PASSWORD=12345
```

Comment:
Change back password to the initial. Initial password was 'root'. If you need your own password you can change it but do not commit it

Reply:
ok, sorry I got it